server/lobbyconnection.py

```diff
@@ -840,13 +840,20 @@ def on_connection_made(self, protocol: QDataStreamProtocol, peername: Address):
         self.player.game = game
         cmd = {
             "command": "game_launch",
-            "mod": game.game_mode,
+            "args": ["/numgames", self.player.game_count[RatingType.GLOBAL]],
             "uid": game.id,
-            "args": ["/numgames " + str(self.player.game_count[RatingType.GLOBAL])]
+            "mod": game.game_mode,
+            "mapname": use_map,
+            "name": game.name,
+            "team": game.get_player_option(self.player, "Team"),
+            "faction": game.get_player_option(self.player, "Faction"),
+            "expected_players": len(game.players) or None,
+            "map_position": game.get_player_option(self.player, "StartSpot"),
+            "init_mode": game.init_mode.value,
```

Comment:
I'd add a comment that those are not used yet so nobody is confused why changing them doesn't do anything.